Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'rake'
 gem 'rspec', '~> 3.11'
 gem 'rubocop-performance', '~> 1.7'
 gem 'rubocop-rake', '~> 0.6'
+gem 'simplecov'
```

Comment:
While we're at it, have you tried https://github.com/grosser/single_cov ?
I have no experience with it, and have no idea if it provides such a pretty report as Simplecov.

Reply:
@ydah can you add minimal coverage?

Reply:
The following was going to need to be added to all specs.

```ruby
require 'spec_helper'
SingleCov.covered!
```

If there are areas not covered by the test as shown below, the following output is likely to be present.

```console
lib/rubocop/cop/rspec/align_left_let_brace.rb new uncovered lines introduced (1 current vs 0 configured)
Lines missing coverage:
lib/rubocop/cop/rspec/align_left_let_brace.rb:25
```